Sources/SwiftGRPC/Runtime/ServiceClient.swift

```diff
@@ -60,7 +60,25 @@ open class ServiceClientBase: ServiceClient {
     self.channel = channel
     metadata = Metadata()
   }
-  
+
+  /// Create a client with Google credentials.
```

Comment:
nit: add "suitable for connecting to a specific Google-provided API", to make it more clear that this can only be used in conjunction with Google services?